### PR TITLE
Fix Mish and SoftPlus value propagation functions

### PR DESCRIPTION
**Pull request by achetver**
Description: Fix for broken value propagation functions for SoftPlus and Mish operations

JIRA: 38592

Code:
* [x]  Comments
* [x]  Code style (PEP8)
* [x]  Transformation generates reshape-able IR
* [x]  Transformation preserves original framework node names

Validation:
* [x]  Unit tests: Done;
* [x]  Framework operation tests: 35814 for SoftPlus operation; 
* [x]  Transformation tests: Not needed, no new implemented transformations;
* [x]  e2e model test with an update of ./tests/e2e_oss/utils/reshape_utils.py: Not needed, no new implemented models;
* [x]  Model Optimizer IR Reader check: Not needed, changes not affect IR;

Documentation:
* [x]  Supported frameworks operations list: Not needed, no new supported operations;
* [x]  Supported **public** models list: Not needed, no new supported models;
* [x]  New operations specification: Not needed, no new operations added;
* [x]  Guide on how to convert the **public** model: Not needed, no new supported models;
* [x]  User guide update: Not needed,nothing to update;

### CODE REVIEW

File: model-optimizer/extensions/ops/activation_ops.py [hunk 1]
```diff
@@ -233,13 +233,13 @@ class Log(Activation):
 class SoftPlus(Activation):
     op = 'SoftPlus'
     version = 'opset4'
-    operation = staticmethod(lambda x: np.ln(np.exp(x) + 1.0))
+    operation = staticmethod(lambda x: np.log(np.exp(x) + 1.0))
```
Comment (lazarevevgeny) [inline model-optimizer/extensions/ops/activation_ops.py:236]: Please, add unit tests for the value propagation functions

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

File: model-optimizer/extensions/ops/activation_ops.py (hunk 1 above)
Comment (achetver) [reply model-optimizer/extensions/ops/activation_ops.py:236]: Tests for SoftPlus & Mish operations added

**Comment by lazarevevgeny** [comment]
@achetver , please, update the ticket description according to template.

File: model-optimizer/extensions/ops/activation_test.py [hunk 2]
```diff
@@ -67,3 +68,63 @@ def test_activation_elu_infer(self):
...
+                                    'operation': SoftPlus.operation,
+                                },
+                                'node_3': {
+                                    'value': None
+                                }
+                            })
+        graph.graph['layout'] = 'NCHW'
+        activation_node = Node(graph, 'activation_node')
+        SoftPlus.infer(activation_node)
+        exp_shape = np.array([227, 227, 227, 227])
```
Comment (lazarevevgeny) [inline model-optimizer/extensions/ops/activation_test.py:93]: Why the expected shape is `[227, 227, 227, 227]`? It should be `[4]` according to the value provided. Please, fix other tests in this file.

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]
Please, fix unit tests

**Review by lazarevevgeny — APPROVED** [review]

**Comment by lazarevevgeny** [comment]
@achetver , please, rebase with master and the pre-commit will pass.

**Pull request merged**